Source/SVGIconVirtualImageList.pas

```diff
@@ -225,6 +220,9 @@ procedure TSVGIconVirtualImageList.SetCollection(const value: TSVGIconImageColle
   if FCollection <> Value then
   begin
     FCollection := Value;
+    if FCollection <> nil then
+      FCollection.FreeNotification(Self);
+
```

Comment:
```suggestion
    if FCollection <> nil then
      FICollection.RemoveFreeNotification(Self);
    FCollection := Value;
    if FCollection <> nil then
      FICollection.FreeNotification(Self);
```

Reply:
hmm.. first time I have used the github commit suggestion and it messed up the merge.